bakery/service/bakery.go

```diff
@@ -75,6 +76,44 @@ func InitConfiguration(configFile, rootDir string) error {
 	return nil
 }
 
+// DumpLatestVersions saves images with their latest version in json format to file.
+// Optionally it can exclude images from provided directories.
+func DumpLatestVersions(fileName, excludeDirsPattern string) error {
+	filterOutImagesFromDirs(excludeDirsPattern)
+	filterOutNotExistingImages()
+	return commons.WriteToJSONFile(versions, fileName)
+}
+
+// filterOutImagesFromDirs removes images that are stored in directories that match provided pattern.
+func filterOutImagesFromDirs(excludeDirsPattern string) {
+	if len(excludeDirsPattern) > 0 {
```

Comment:
`if len(excludeDirsPattern) <= 0 { return}`  would be more idiomatic go

Reply:
Done